{"uid":"4a67542dce4b5315","name":"Logout of system test","fullName":"tests.login_test.TestLogin#test_logout","historyId":"e89a5a0e9aefe451ee262eba82370b01","time":{"start":1748483613874,"stop":1748483613874,"duration":0},"description":"Log out from app","descriptionHtml":"<p>Log out from app</p>\n","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1748483587467,"stop":1748483587467,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1748483587467,"stop":1748483587468,"duration":1},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/conftest.py\", line 60, in data\n    with open(json_path, encoding=\"utf-8\") as json_file:\n         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"base_url","time":{"start":1748483587467,"stop":1748483587467,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"session_request","time":{"start":1748483587467,"stop":1748483587467,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1748483612112,"stop":1748483612112,"duration":0},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/conftest.py\", line 60, in data\n    with open(json_path, encoding=\"utf-8\") as json_file:\n         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Log out from app","status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","statusTrace":"@fixture(scope=\"session\")\n    def data() -> Data:\n        json_path = Constants.DATA_PATH / \"tests_data.json\"\n>       with open(json_path, encoding=\"utf-8\") as json_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'\n\ntests/conftest.py:60: FileNotFoundError","steps":[{"name":"public ip address","time":{"start":1748483616923,"stop":1748483616927,"duration":4},"status":"passed","steps":[],"attachments":[{"uid":"26dabff39e7b0bc","name":"HTTP logs of http://checkip.amazonaws.com/","source":"26dabff39e7b0bc.txt","type":"text/plain","size":357}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"69a49ff765b97c59","name":"Cookies","source":"69a49ff765b97c59.json","type":"application/json","size":4118},{"uid":"f8e19355bba45aee","name":"Session Storage","source":"f8e19355bba45aee.json","type":"application/json","size":359},{"uid":"3d6bcd6dcfee3533","name":"Local Storage","source":"3d6bcd6dcfee3533.json","type":"application/json","size":548},{"uid":"2a124c5058ce469b","name":"Full Page Screenshot","source":"2a124c5058ce469b.png","type":"image/png","size":1363102},{"uid":"5757de161ea7fa4e","name":"URL","source":"5757de161ea7fa4e.uri","type":"text/uri-list","size":23},{"uid":"8ebe70874da7b681","name":"Network Logs","source":"8ebe70874da7b681.json","type":"application/json","size":48962}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":7,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"session_request::0","time":{"start":1748483618497,"stop":1748483618498,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"story","value":"As a user I want to be able to logout after a successful login."},{"name":"severity","value":"blocker"},{"name":"feature","value":"Login"},{"name":"epic","value":"Security"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1068-680"},{"name":"thread","value":"2059-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":19,"skipped":0,"passed":0,"unknown":0,"total":19},"items":[{"uid":"1b17244638ff4753","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/667//#testresult/1b17244638ff4753","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748397239924,"stop":1748397239924,"duration":0}},{"uid":"4e5d3f643655c02a","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/666//#testresult/4e5d3f643655c02a","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748310700532,"stop":1748310700532,"duration":0}},{"uid":"265cc28363981bf9","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/665//#testresult/265cc28363981bf9","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748224567051,"stop":1748224567051,"duration":0}},{"uid":"f12f8bfd20606fda","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/664//#testresult/f12f8bfd20606fda","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748138483914,"stop":1748138483914,"duration":0}},{"uid":"20dffdc7ce684e8b","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/663//#testresult/20dffdc7ce684e8b","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1748051348868,"stop":1748051348868,"duration":0}},{"uid":"bb77f9109fcb09c","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/661//#testresult/bb77f9109fcb09c","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747878757441,"stop":1747878757441,"duration":0}},{"uid":"b41342851159e50f","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/660//#testresult/b41342851159e50f","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747792399602,"stop":1747792399602,"duration":0}},{"uid":"e07e367b33b5c9c5","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/659//#testresult/e07e367b33b5c9c5","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747706023231,"stop":1747706023231,"duration":0}},{"uid":"bfd1be004dbaa55f","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/658//#testresult/bfd1be004dbaa55f","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747619855958,"stop":1747619855958,"duration":0}},{"uid":"348e585c9aad72a6","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/657//#testresult/348e585c9aad72a6","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747533520891,"stop":1747533520891,"duration":0}},{"uid":"8882d12c5fe72847","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/656//#testresult/8882d12c5fe72847","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747446581473,"stop":1747446581473,"duration":0}},{"uid":"3a2816b58c81413d","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/655//#testresult/3a2816b58c81413d","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747360395296,"stop":1747360395296,"duration":0}},{"uid":"d601a33ae5e9d574","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/654//#testresult/d601a33ae5e9d574","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747273797979,"stop":1747273797979,"duration":0}},{"uid":"58e0765270314b81","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/653//#testresult/58e0765270314b81","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747187502062,"stop":1747187502062,"duration":0}},{"uid":"f6e7445de423bbda","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/652//#testresult/f6e7445de423bbda","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747101185656,"stop":1747101185656,"duration":0}},{"uid":"8521f2af2e39ff8","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/651//#testresult/8521f2af2e39ff8","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1747014936544,"stop":1747014936544,"duration":0}},{"uid":"c370229aa47ac709","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/650//#testresult/c370229aa47ac709","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1746928626044,"stop":1746928626044,"duration":0}},{"uid":"62145f41bc161ec9","reportUrl":"https://nirtal85.github.io/Selenium-Python-Example/649//#testresult/62145f41bc161ec9","status":"broken","statusDetails":"FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/src/resources/tests_data.json'","time":{"start":1746841613459,"stop":1746841613459,"duration":0}}]},"tags":["security"]},"source":"4a67542dce4b5315.json","parameterValues":[]}